### ISSUE TITLE
compiler ignores checkTypes by default

### ISSUE THREAD

**Issue by ikabirov** [google/closure-compiler#2171]
when i compile code
```
/** @type {string} */
var s = 1;
```
with command line: `java -client -jar closure-compiler-v20161024.jar --js main.js --compilation_level=ADVANCED_OPTIMIZATIONS --warning_level=DEFAULT`
there are no warnings

I can't use `--warning_level=VERBOSE` due to https://github.com/google/closure-compiler/issues/2136

**Comment by ChadKillingsworth** [comment]
All of the checks may be individually toggled with the `--jscomp_off`,  `--jscomp_warn` and `--jscomp_error` flags. 

The warning levels are just convenient presets.

**Comment by ikabirov** [comment]
But wiki https://github.com/google/closure-compiler/wiki/Warnings says than checkTypes must be WARNING by default. 
Moreover some checks (eg accessControls) doesn't work with --warning_level=DEFAULT

**Comment by dimvar** [comment]
@ikabirov Typechecking is on by default in advanced mode. I'm not sure why you are not seeing an error in your environment. Maybe you're somehow redirecting stderr or something.

https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%252F%252F%2520ADD%2520YOUR%2520CODE%2520HERE%250A%250A%252F**%2520%2540type%2520%257Bstring%257D%2520*%252F%250Avar%2520s%2520%253D%25201%253B%250A

**Comment by ikabirov** [comment]
@dimvar incorrect sample, online compiler uses warning_level=verbose by default

We can see that in POST data tab:
```
output_format=json
&output_info=compiled_code
&output_info=warnings
&output_info=errors
&output_info=statistics
&compilation_level=ADVANCED_OPTIMIZATIONS
&warning_level=verbose
&output_file_name=default.js
&js_code=%2F%2F%20ADD%20YOUR%20CODE%20HERE%0A%0A%2F**%20%40type%20%7Bstring%7D%20*%2F%0Avar%20s%20%3D%201%3B
```

**Comment by dimvar** [comment]
You're right. There is a mismatch in advanced mode between command-line and appspot. I'll fix.

**Comment by fredj** [comment]
fixed with 085928c
